docs/source/model-registry.rst

```diff
@@ -16,31 +16,31 @@ staging to production), and annotations.
 Concepts
 ========
 
-The MLflow Model Registry is a standalone component that introduces several concepts that
-describe the full lifecycle of an MLflow Model.
+As the fourth standalone component of MLflow, the Model Registry introduces few concepts that describe and facilitate the full lifecycle of an MLflow model.
```

Comment:
```suggestion
As the fourth standalone component of MLflow, the Model Registry introduces a few concepts that describe and facilitate the full lifecycle of an MLflow model.
```

Reply:
Is there a reason why we are highlighting that this is the "fourth" component?

Reply:
More from having had three components for a while and having MLflow touted as component-based and modularized, thought we would highlight it. The next or 5th pillar might be Model Serving. I

Reply:
Thinking more about it, "standalone" may be misleading or ambiguous as much as I wish to convey its modularity, and "fourth" I'm not wedded to it. We can drop both, and just rephrase as
"As a new component of MLflow," or just drop the introductory phrase. Let you decide.

Reply:
I vote for dropping the phrase.